.circleci/config.yml

```diff
@@ -348,15 +359,42 @@ jobs:
       package_manager:
         type: enum
         enum: ["npm", "npm_global", "pnpm"]
-    executor: node_js
+      platform:
+        type: enum
+        enum: ["nix", "windows"]
+    executor: node_js_<<parameters.platform>>
     steps:
       - checkout:
           path: "rover"
-      - run:
-          name: "Invoke Install Scripts"
-          command: |
-            cd rover/.circleci/scripts
-            ./install_<< parameters.package_manager >>.sh
+      - when:
+          condition:
+            equal: ["nix", <<parameters.platform>>]
+          steps:
+            - run:
+                name: "Invoke Install Scripts (Unix)"
+                command: |
+                  cd rover/.circleci/scripts/<<parameters.platform>>
+                  ./install_<< parameters.package_manager >>.sh
+      - when:
+          condition:
+            equal: ["windows", <<parameters.platform>>]
+          steps:
+            - run:
+                name: "Invoke Install Scripts (Windows)"
+                command: |
+                  Write-Output "Installing Volta"
+                  choco install volta
+                  refreshenv
+                  Write-Output "Installing Node & NPM"
+                  volta install node@20.16.0
+                  Write-Output "Checking Node & NPM version"
+                  node --version
+                  npm --version
```

Comment:
Annoyingly the Windows Executors come bundled with `nvm`, `node` and `npm` but they're so old they don't work and the `nvm` install is fundamentally broken. So we have to do a bit of setup here

Reply:
Well that is fun!